TestCases/disc_adj_rans/cylinder_DT_1ST/cylinder.cfg

```diff
@@ -36,6 +36,11 @@ WRT_CSV_SOL= NO
 WRT_VOL_SOL= NO
 %
 WRT_SRF_SOL= NO
+
+SINGLEZONE_DRIVER = YES
```

Comment:
I cant find any of these 4 new config options in the config_template. Please add those with a description of how and when to use it.

Reply:
The idea is that SINGLEZONE_DRIVER goes away after the driver refactoring is complete, that's why it wasn't added so far. I've just tentatively added them with a brief explanation, but further description will be needed once the driver is completely refactored and ITER / TIME_ITER become the standard.